Barotrauma/BarotraumaClient/ClientSource/Sprite/DeformAnimations/CustomDeformation.cs

```diff
@@ -109,9 +109,28 @@ public override float Phase
             }
         }
 
-        protected override void GetDeformation(out Vector2[,] deformation, out float multiplier, bool inverse)
+        protected override void GetDeformation(out Vector2[,] deformation, out float multiplier, bool inverse, bool isFlipped = false)
         {
-            deformation = Deformation;
+            if (isFlipped)
+            {
+                Vector2[,] flippedDeformation = new Vector2[Deformation.GetLength(0), Deformation.GetLength(1)];
+
+                int width = Deformation.GetLength(0);
+                int height = Deformation.GetLength(1);
```

Comment:
Would make sense to store the width and height before creating the array and reuse the values, instead of getting the lengths twice.